swo/mpt/cli/core/accounts/flows.py

```diff
@@ -0,0 +1,104 @@
+import json
+import os
+from operator import attrgetter
+from pathlib import Path
+
+from swo.mpt.cli.core.accounts.models import Account
+from swo.mpt.cli.core.errors import AccountNotFoundError
+from swo.mpt.cli.core.mpt.models import Token
+
+
+def from_token(token: Token, secret: str, environment: str) -> Account:
+    """
+    Extracts Account from the MPT Token
+    """
+    return Account(
+        id=token.account.id,
+        name=token.account.name,
+        type=token.account.type,
+        secret=secret,
+        environment=environment,
+        is_active=True,
```

Comment:
I would suggest to just call it `active`. The `is_` prefix makes me think on a function.